app/views/add-reduction.html

```diff
@@ -0,0 +1,168 @@
+{% extends "includes/layout.html" %} {% block content %}
+
+<!-- TEMPORARY VALUE START -->
+{% set isViewOnly=false %}
+
+<!-- TEMPORARY VALUE END -->
+
+<main id="content" role="main">
+  {% include "includes/breadcrumbs.html" %}
+  
+  {% if isViewOnly %}
```

Comment:
Maybe a different name for this var, like `isAdd`? Since add is the different case, whereas the screen will then look the same for view, edit, archive, delete